apps/agentic-chat/src/types/asset.ts

```diff
@@ -1,10 +1,7 @@
-import { AssetId, ChainId } from '@shapeshiftoss/caip';
+import { Asset as FullAsset } from '@shapeshiftoss/types';
 
-export type Asset = {
-  assetId: AssetId;
-  chainId: ChainId;
-  symbol: string;
-  name: string;
-  precision: number;
-  icon: string | undefined;
-};
+// Minimal Asset type definition not to make things more complicated than they need to be for this repo
```

Comment:
🧠

Reply:
So far that'd be the only one, may or may not need more in the future though. Happy to revert and redeclare `Asset` type here (i.e the minimal asset type here) if we feel like this is fluff add, pretty agnostic since types are not shipped at runtime anyway, happy with both ways!

Reply:
I say remove until if/when we need it for any reason haha